spec/rails_helper.rb

```diff
@@ -0,0 +1,76 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
not sure what the difference is between ```rails_helper.rb``` and ```spec_helper.rb``` or if i need both but I configured capybara/selenium on here.